src/game/save_file.h

```diff
@@ -62,16 +62,12 @@ struct MainMenuSaveData {
 #ifdef WIDE
     u8 wideMode: 1;
 #endif
-
-#if MULTILANG
-    u8 language: 2;
-#define SUBTRAHEND 8
-#else
-#define SUBTRAHEND 6
+#ifdef MULTILANG
+    u8 language: 3;
 #endif
-    u8 firstBoot;
 
     #ifdef PUPPYCAM
```

Comment:
While you're here, please fix ifdef indentation thanks

Reply:
lmao